common/models/resource.go

```diff
@@ -0,0 +1,9 @@
+package models
+
+type Resource struct {
```

Comment:
Not sure why this is a model - we don't use these objects to display anything to users

Reply:
Good point. Maybe it's a holdover from when we talked (?) about having the scaler be a standalone thing? Regardless, I agree, these structs should be moved locally.

Reply:
IIRC (and @nehayward please correct me if I'm wrong), but this was part of an experiment to see if it were absolutely necessary for `ResourceProvider` and `ResourceConsumer` to be two different things. I think the conclusion was that yes, they do need to be separate, and so this would want to go away.

Reply:
Now I understand what you all meant from a comment over in #440 - I'll make this change over there.

✅

Reply:
Yup it was. @tlake.